brianlong/www.validators.app
# 182536077 live update pt table

#### What's this PR do?
- reduce the number of ping things in the table
- remove pagination
- live update for table

#### How should this be manually tested?
- add ping things via console or api and check if the table updates as well
- check if the number of records is reduced in the table

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182536077)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
